### PR TITLE
Added card deck synchronization

### PR DESCRIPTION
**Pull request by mdangnguyen**
Fixed page reloading bug

### CODE REVIEW

File: digital-deck-ui/src/store/index.js [hunk 1]
```diff
@@ -66,15 +66,15 @@ export default createStore({
...
                 commit('setPlayerCards', res.data.players[0].cards);
                 commit('setTableCards', res.data.table);
-                commit('setCardsInDeck', res.data.deck);
+                commit('setCardsInDeck', res.data.deck.length);
                 commit('setName', res.data.players[0].name);
                 commit('setPlayersInfo', res.data.players);
                 $cookies.set('SessionId', res.data.sessionId, '1h');
                 UnitTests.testInitSession(state);
-                return true;
+                // return true;
```
Comment (suryarawat) [inline digital-deck-ui/src/store/index.js:74]: you can remove this

File: digital-deck-ui/src/components/MainMenu.vue [hunk 2]
```diff
@@ -149,16 +146,16 @@ export default {
...
             decks: this.deckSelected,
             name: this.name,
             cardsPerPlayer: this.cardsPerPlayer,
             cardsOnTable: this.cardsOnTable,
-          })
-        if (success) {
-          this.isLoaded = true;
-        }
+          }).then(() => {
+            console.log(this.$store.getters.getSessionId);
```
Comment (suryarawat) [inline digital-deck-ui/src/components/MainMenu.vue:156]: You can remove this before pushing

File: digital-deck-server/socket.js [hunk 3]
```diff
@@ -6,9 +6,17 @@ function handleSocket(io)
...
+            console.log("in server", sessionId);
+        });
+
+        socket.on('disconnect', () => {
+            console.log("Disconnected");
+        });
+
+        socket.on('drawCard', ({ sessionId, numCards, player }) => {
+            // console.log("current cards", cards);
+            socket.to(sessionId).emit('cardDrawn', { deck: numCards, player: player });
```
Comment (suryarawat) [inline digital-deck-server/socket.js:18]: Can you please confirm that integers will work as room names?

Comment (mdangnguyen) [reply digital-deck-server/socket.js:18]: It does. I'm pretty sure Hassan also confirmed it when working on his ticket

**Review by suryarawat — APPROVED** [review]

**Pull request merged**